network/evpn-gw/evpn-gw-cni/pkg/utils/netlink_manager.go

```diff
@@ -0,0 +1,88 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright (C) 2023 Network Plumping Working Group
+// Copyright (C) 2023 Nordix Foundation.
+
+package utils
```

Comment:
code dup with evpn-gw bridge, just import that package
https://github.com/opiproject/opi-evpn-bridge/blob/c0c75a9b324f8795426abfa608993fe696e6427c/pkg/utils/netlink.go